styles/signals.mapcss

```diff
@@ -51,9 +51,9 @@ way[!"construction:railway"]["construction"=rail]["construction:railway:etcs"]["
 	}
 }
 
-way["railway:pzb"=no]["railway:lzb"=no][!"railway:etcs"].tracks,
-way["railway:pzb"=no]["railway:lzb"=no]["railway:etcs"=no].tracks,
-way["railway:pzb"=no]["railway:lzb"=no]["railway:etcs"=0].tracks
+way["railway:pzb"=no]["railway:lzb"=no]["railway:atb"=no][!"railway:etcs"].tracks,
```

Comment:
This will not work:I would bet there is no single track in OSM that is tagged all with lzb=no, pzb=no, and atb=no. Keep the old lines, and add 3 new ones with atb=no and the etcs variants.

Reply:
done (I hope...)

Reply:
All tracks in the Netherlands without ATB contain the following tags:
railway:atb=no
railway:etcs=no
railway:pzb=no
railway:lzb=no
railway:gnt=no

Example changeset:
https://www.openstreetmap.org/changeset/66868052#map=9/53.0389/5.5528
Example on ORM:
https://www.openrailwaymap.org/mobile.php?lang=null&lat=51.53984933848655&lon=4.45871114730835&zoom=15&style=signals